### ISSUE TITLE
[From Android team] Improved issue management tracking via Oppiabot

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppiabot#355]
There may be some nice triage improvements we could get from Oppia bot:
1. Making sure that all of the key categories of labels are specified for issues
2. Ensure a milestone & project is properly defined for all issues labeled as 'Type: task'
3. Ensure when either a milestone or project is defined, the opposite is also defined
4. Track target completion date (e.g. via a 'target date' in the main comment) and comment on the issue when this date is exceeded
5. Automatically change the 'status' of an issue based on its presence in a project & other state (like blocking status--see below)
6. Enforce blocking statuses for issues
7. Ensure fixed issues are marked as fixed for its status
8. Ensure issues have an enforced verification pipeline
9. Ensure closed & finished statuses correspond correctly to whether the issue itself is actually closed
10. Maybe: enforce that the issue is in-implementation if there's a corresponding pull request linked to it
11. Enforce that PRs are set to be merged into develop, or marked as blocked on another PR that’s either marked as blocked or based on develop. This guards against longterm feature branches, and we can add a check that intentionally fails to block merging these PRs until they’re based on develop. 
12. Enforce that PRs have branches with names that follow our conventional branch name: ``name-of-branch``
